env/common/files/system/logrotate.syslog

```diff
@@ -13,8 +13,12 @@
 /var/log/messages {
     rotate 30
     daily
+    compress
+    compresscmd /usr/bin/gpg2
+    compressoptions --encrypt -r "Sys Admin" --homedir /root/.gnupg
+    compressext .gpg
+
     postrotate
       /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-      /usr/local/bin/log_strip_out_uwsgi.py -r $1-`date --date="yesterday"  '+%Y%m%d'`
```

Comment:
If this script is not being used, it should also be removed from the task-playbook? https://github.com/usegalaxy-no/infrastructure-playbook/blob/fc71c6a7898e13128b8e685944e66bc44447a213/env/common/tasks/logrotate.yml#L65

Reply:
yes, you're right. I wasn't sure if it was the only use case for this script. I'll remove it tomorrow and re-request the merge.